tests/10-vcdm2.js

```diff
@@ -317,6 +317,13 @@ describe('Verifiable Credentials Data Model v2.0', function() {
         await assert.rejects(
           issue(require('./input/credential-issuer-object-no-id-fail.json')));
       });
+      it2('If present, the value of the "issuer.description" property ' +
+      'MUST be a string or a language value object as described in 10.1 ' +
+      'Language and Base Direction.', async function() {
+        await issue(require('./input/credential-issuer-description-ok.json'));
+        await issue(require(
+          './input/credential-issuer-description-language-en-ok.json'));
```

Comment:
Do we have a test for issuer with no description property and it passes? If we don't we should add it here.

Reply:
we have like 30-40 tests where the issuer has no description, but I can add those in a moment.

Reply:
added tests for optional description: https://github.com/w3c/vc-data-model-2.0-test-suite/pull/48/commits/a131baa811052f3021ae262e5708b0f2b9917d03 and for optional name: https://github.com/w3c/vc-data-model-2.0-test-suite/pull/48/commits/cdcee49926fd4678e424eff70307ca23c039462f